README.md

```diff
@@ -1,111 +1,160 @@
 
-    git clone https://github.com/chanzuckerberg/cellxgene.git
+There are several options available, such as:
 
-**Install client**
+- `--layout` to specify the layout as  `tsne` or `umap`
+- `--title` to show a title on the explorer
+- `--open` to automatically open the web browser after launching
 
-    cd cellxgene
-    ./bin/build-client
+To see all options call
 
-**To use with virtual env for python**
-(optional, but recommended)
+```
+cellxgene launch --help
+```
 
-    ENV_NAME=cellxgene
-    python3 -m venv ${ENV_NAME}
-    source ${ENV_NAME}/bin/activate
+## data formatting
 
-**Install server**
+### assumptions
 
-     pip install -e .
+The `launch` command assumes that the data is stored in the `.h5ad` format from the [`anndata`](https://anndata.readthedocs.io/en/latest/index.html) library. It also assumes that certain computations have already been performed. Briefly, the `.h5ad` format wraps a two-dimensional `ndarray` and stores additional metadata as "annotations" for either observations (referred to as `obs` and `obsm`) or variables (`var` and `varm`). `cellxgene launch` makes the following assumptions about your data (we recommend loading and inspecting your data using `scanpy` to validate these assumptions)
 
-**Run (with demo data)**
+- an `obs` field has a unique identifier for every cell (you can specify which field to use with the `--obs-names` option, by default it will use the value of `data.obs_names`)
+- a `var` field has a unique identifier for every gene (you can specify which field to use with the `--var-names` option, by default it will use the value of `data.var_names`)
+- an `obsm` field contains the two-dimensional coordinates for the layout that you want to render (e.g. `X_tsne` for the `tsne` layout or `X_umap` for the `umap` layout)
+- any additional  `obs` fields will be rendered as per-cell continuous or categorical metadata by the app (e.g. `louvain` cluster assignments) 
 
-    cellxgene launch --title PBMC3K example-dataset/pbmc3k.h5ad
+### prepare
 
-**Help**
+The `prepare` command is included to help you format your data. It uses `scanpy` under the hood. This is especially useful if you are starting with raw unanalyzed data and are unfamiliar with `scanpy`. 
 
-    cellxgene --help
+To prepare from an existing `.h5ad` file use
 
-_For help with the scanpy engine_
+```
+cellxgene prepare dataset.h5ad --output=dataset-processed.h5ad
+```
+
+This will load the input data, perform PCA and nearest neighbor calculation, compute `umap` and `tsne` layouts and `louvain` cluster assignments, and save the results in a new file called `dataset-processed.h5ad` that can be loaded using `cellxgene launch`. Data can be loaded from several formats, including `.h5ad` `.loom` and a `10-Genomics-formatted` `mtx` directory. Several options are available, including running one of the preprocessing `recipes` included with `scanpy`, which include steps like cell filtering and gene selection. To see all options call
```

Comment:
Suggest adding to the FAQ several examples/recipes:
* starting with the default 10X file (cell ranger output), step-by-step list of prepare steps & display steps
* ditto for other common formats

Reply:
take a look at what i added and see if that's what you had in mind?